### PR TITLE
mpd: Add version 0.23.7

### PR DESCRIPTION
**Pull request by bharatvaj**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/mpd.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "0.23.7",
+    "description": "Music Player Daemon",
+    "homepage": "https://www.musicpd.org/",
+    "license": "GPL-2.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://www.musicpd.org/download/win32/0.23.7/mpd.exe",
+            "hash": "7e30ceb29544039a94b2b234901403ffa6c984cb0c8248c1f596314d203c8c2f"
+        },
```
Comment (goostleek) [inline bucket/mpd.json:10]:
```suggestion
        }
```

Comment (bharatvaj) [reply bucket/mpd.json]: Done @goostleek

File: bucket/mpd.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "64bit": {
+            "url": "https://www.musicpd.org/download/win32/0.23.7/mpd.exe",
+            "hash": "7e30ceb29544039a94b2b234901403ffa6c984cb0c8248c1f596314d203c8c2f"
+        }
+    },
+    "bin": "mpd.exe",
+    "checkver": {
+        "github": "https://github.com/MusicPlayerDaemon/MPD",
+        "regex": "tag/v([\\d.]+)"
+    },
```
Comment (rashil2000) [inline bucket/mpd.json:16]:
```suggestion
    "checkver": "MPD ([\\d.]+) released",
```

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**